widgetssdk/src/main/java/com/glia/widgets/helper/Files.kt

```diff
@@ -81,40 +87,49 @@ internal fun fixCapturedPhotoRotation(uri: Uri, context: Context) {
     }
 }
 
-internal fun getContentUriCompat(fileName: String, context: Context): Uri =
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
-        val downloadsContentUri = MediaStore.Downloads.getContentUri(MediaStore.VOLUME_EXTERNAL)
-        val projection = arrayOf(
-            MediaStore.Downloads._ID,
-            MediaStore.Downloads.DISPLAY_NAME,
-            MediaStore.Downloads.SIZE
-        )
-        val selection = MediaStore.Downloads.DISPLAY_NAME + " == ?"
-        val selectionArgs = arrayOf(fileName)
-        val sortOrder = MediaStore.Downloads.DISPLAY_NAME + " ASC"
-
-        context.contentResolver.query(
-            downloadsContentUri,
-            projection,
-            selection,
-            selectionArgs,
-            sortOrder
-        )?.use {
-            if (it.count == 0) return@use Uri.EMPTY
-
+@RequiresApi(Build.VERSION_CODES.Q)
+private fun getContentUriAPI29(fileName: String, context: Context): Uri? {
```

Comment:
NIT, not critical at all, related to style guide.
I think the name should be `getContentUriApi29()`

Reply:
fixed c564b40cc7d5f30ec018a0225b08b5e33b5ef236